src/components/editor/EditorArticleController.jsx

```diff
@@ -92,8 +106,55 @@ export default class EditorArticleController extends FalcorController {
     }, 2000);
   }
 
+  // Handle state changes coming from editAuthorsForm
+
+  handleAddAuthor(slug) {
+    // TODO: Autocomplete which also includes validating that the authors indeed exist
+
+    // Is this author already added?
+    if (!slug || this.state.authorsAdded.hasOwnProperty(slug) || this.state.authorsDeleted.hasOwnProperty(slug)) {
+      return;
+    }
+    let newValue = {};
+    // Remember to change value of this to slug or however we end up storing
+    newValue[slug] = slug;
```

Comment:
But I want the be variable. Not to be "slug" but be the actual slug. Also doing extra research they agree with me. http://stackoverflow.com/questions/11508463/javascript-set-object-key-by-variable So I think that's the only way